### ISSUE TITLE
About the SCANLoss

### ISSUE THREAD

**Issue by suzhenghang** [wvangansbeke/Unsupervised-Classification#24]
the SCANLoss  in losses.py is 
total_loss = consistency_loss - self.entropy_weight * entropy_loss, however it is diferent from paper 
![image](https://user-images.githubusercontent.com/8575329/90112521-af044100-dd82-11ea-9e96-6eeeb71d3aa8.png)

**Comment by wvangansbeke** [comment]
Hi @suzhenghang,

Why do you think it is different? In equation 2, the first term is the consistency loss and the second term is the negative entropy (the entropy is -p*log(p)). There are only two practical differences:
- Instead of using the whole dataset D, the loss is computed over a batch of images.
- During training we uniformly sample 1 random neighbor at a time. This allows us to scale up to a large amount of neighbors (see issue #4).

If you have some specific question, let me know.

**Comment by wvangansbeke** [comment]
Hi @suzhenghang,

I guess everything is clear now? If not, feel free to ask more questions. I'm closing this issue for now.
